extensions/amp-list/0.1/amp-list.js

```diff
@@ -352,10 +351,7 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   isAmpStateSrc_(src) {
-    return (
-      isExperimentOn(this.win, 'amp-list-init-from-state') &&
-      startsWith(src, AMP_STATE_URI_SCHEME)
-    );
+    return startsWith(src, AMP_STATE_URI_SCHEME);
```

Comment:
Typically we'd first "launch" the experiment by setting the value in canary-config.json and prod-config.json to 1. This allows for reduced risk of merge conflicts if we need to rollback.

Reply:
Done 👍